### PR TITLE
Feature/27 update business event model

### LINKED ISSUE #27: Update Business Event Model
The data model for a business event has changed. See the latest version [here](https://techwiki.zalando.net/pages/viewpage.action?spaceKey=DT&title=Project+Honeypot).

We need to update our class & interface to match the new model.

### PR DESCRIPTION
**Pull request by frontendherodk**
Closes #27

### CODE REVIEW

File: src/tracking.service.ts [hunk 1]
```diff
@@ -20,10 +20,20 @@ class ScTrackingService {
...
+  private constructUrl(eventType: string): string {
+    return `${this.eventUrl}${eventType}`;
+  }
+
+  track(eventType: string, e: EventInterface): ng.IPromise<any> {
+    if (!eventType) {
+      return this.$q.reject(new TrackingError(TrackingError.NO_EVENT_TYPE));
+    }
+
+    let endpointUrl = this.constructUrl(eventType);
```
Comment (cjhowald) [inline src/tracking.service.ts]: (nitpicky)
Do we need this variable if we only use it once 4 lines below? Why not just construct the url on that line?

**Comment by cjhowald** [comment]
👍

**Comment by cjhowald** [comment]
@dami-gg can you review this and 👍  if it looks good?

**Comment by dami-gg** [comment]
:+1:

**Pull request merged**